src/main/java/dansplugins/factionsystem/eventhandlers/DamageHandler.java

```diff
@@ -55,11 +55,11 @@ public void handle(EntityDamageByEntityEvent event) {
 
         if (attacker == null || victim == null) {
             logger.debug("Attacker and/or victim was null in the DamageHandler class.");
+            handleEntityDamage(attacker, event);
```

Comment:
Why has this been moved?

Reply:
Because the victim is always a player but when you hit an entity, the victim is null.

Reply:
I see - thank you!